<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Sudhanshus-MacBook-Air.local" failures="1" tests="6" name="Test" time="56.049" errors="0" timestamp="2024-09-29T19:48:13 IST">
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="3.998"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="3.189"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="1.964">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="16.561"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="4.358"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="11.419"/>
</testsuite> <!-- Test -->
